pkg/controller/oneagentapm/oneagentapm_controller_test.go

```diff
@@ -26,21 +26,26 @@ func init() {
 	os.Setenv(k8sutil.WatchNamespaceEnvVar, "dynatrace")
 }
 
-func TestReconcileOneAgentAPM(t *testing.T) {
-	name := "oneagent"
-	ns := "dynatrace"
+const (
+	apiUrl    = "https://ENVIRONMENTID.live.dynatrace.com/api"
+	name      = "oneagent"
+	namespace = "dynatrace"
+
+	noTokenError = "Secret 'dynatrace:oneagent' not found"
```

Comment:
If it's a variable used only once, then I suggest using it directly on the function who calls it.